content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,207 @@
+Clients can create and trade assets on the X-Chain and other examples of the AVM with the help of the X-Chain API.
+
+## Flare's FBA
+
+TODO: add how Flare hijacks the codebase to run FBA on it.
+
+## State Connector
+
+The State Connector – Flare’s system to observe the state of underlying chains.
+State Connector is one of the most critical protocols on the Flare Network.
+The purpose of a state connector is to connect non-flare assets (like XRP, Dode, LTC, etc.) with the Flare Network (represented by F-Assets) in a trustless manner, or other words, allow the smart contracts on Flare to act on state changes from underlying blockchains.
+The State Connector adopts other blockchains’ consensus mechanisms and validator ecosystems.
+
+### Advantages
+
+A state connector system is a competitive approach for proving the state of an underlying chain to a smart contract, and it has the following advantages:
+
+* Transaction validity references back to an underlying chain's genesis block: Other approaches like the SPV (Simplified Payment Verification) proof does not check the validity of a transaction, because it is considered harmful for proving the state of an underlying chain to another network.
+* Safety only depends on an underlying chain's validators: There is no trusted third-party service that has its own set of economic incentives and risks.
+  Trust is minimized by leveraging the guarantee that safety can only be lost in the state connector if an underlying chain's validators encounter a Byzantine fault.
+* No cooperation needed from an underlying chain's validators: Validators from an underlying chain do not need to modify their chain's codebase to permit Flare to interpret their network.
+  An underlying chain's validators do not even need to be aware that Flare exists for the state connector system to operate.
+* Can read the state of any blockchain: The state connector can operate on any possible Sybil-resistance technique of an underlying chain.
+  For example proof-of-work, proof-of-stake, and even federated byzantine agreement where there is no global agreement on the set of validators in control of a network.
+* No encoding of the current validators in control of an underlying chain to a smart contract on Flare: This requirement of other state-relay approaches such as the SPV proof leads to the hazardous scenario where the enforcement of bad behavior in relaying state needs to be conducted by the same set of operators that have performed the bad behavior.
+* Constant-sized proofs: both the data availability proof and the payment proof are constant-sized, independent of the number of other payments in the data availability period being considered.
+* Every Flare validator independently verifies an underlying chain's state: If your Flare validator observes the canonical state of an underlying chain, then you will not lose safety against that chain.
+
+### Design
+
+State Connector is built into the validators.
```

Comment:
```suggestion
The State Connector is built into the validators.
```

Reply:
So attestors (which can also be price providers of FTSO but don't have to be) constitute the state connector system?